doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -0,0 +1,75 @@
+.. _ug_matter_configuring_protocol:
+
+Configuring Matter in |NCS|
```

Comment:
Alternative to consider: "Configuring |NCS| application for Matter"

Reply:
I'll stick to the original, because your proposal is way longer although it is more accurate.